index.html

```diff
@@ -0,0 +1,180 @@
+            design and landscape industry to expand your business in the Middle
+            East, Africa and South Asia. The event offers live product
+            showcases, knowledge exchange, and networking opportunities, all
+            alongside the prestigious Big 5 Global event.
+          </p>
+          <h2>2023.10.16(MON) ~ 19(THU)</h2>
+          <p class="foot-p">
+            @ DMG events is an organiser of face-to-face events and a publisher
+            of trade magazines for the international Construction, Energy,
+            Coatings, Transport, Hospitality & Design industries.
+          </p>
+        </div>
+      </div>
+    </section>
+    <!-- home page ends here -->
+    <!-- cards starts here -->
+    <section id="cards" class="cards">
+      <p>Main Program</p>
+      <hr class="about-hr" />
+      <div class="card-group">
+        <div class="card">
+          <img src="./assests/lecture.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">Lecture</h5>
+            <p class="card-text">
+              Listen to the speakers from the various countries about the
+              messages of sharing and opening
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <img src="./assests/Exhibition.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">CC Exhibition</h5>
+            <p class="card-text">
+              Appreacite various creations applying CC licence of artists,
+              organized from DMG events.
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <img src="./assests/forum.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">Forum</h5>
+            <p class="card-text">
+              Have the time to share your thoughts and opnions with experts for
+              each topic.
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <img src="./assests/workshop.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">Workshop</h5>
+            <p class="card-text">
+              Try creating your own work using open source license rather than
+              just watching it.
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <img src="./assests/ignite.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">CC Ignite</h5>
+            <p class="card-text">
+              Get opportunities to netwrok with CC affiliates around the world,
+              also after the summit.
+            </p>
+          </div>
+        </div>
+      </div>
+      <button type="button">Join Urban Design and Landscape 2023</button>
+    </section>
```

Comment:
- [ ] please add some margin left and right to your desktop speaker section. Also, kindly add the small gray line under each speaker red title. Remember keeping an eye to small details is good for design.
![Screenshot (1202)_LI](https://github.com/Kaiserabbas/Capstone/assets/65298048/b28fac20-d4ac-4f4b-938f-b957610beeae)